development-defaults.ini

```diff
@@ -1,5 +1,5 @@
 
-path = "/uber"
```

Comment:
> While I personally would prefer MAGFest's system to also use the 'rams' prefix, I suspect that people who have memorized URLs will hang me if I do that.

Possible solution: set up our Nginx web servers to automatically redirect all URLs that start with ``/uber`` to exactly the same URL but starting with ``/rams`` instead.  Everyone's bookmarks and memories will still work as they expect with no breakage.